bedrock/products/templates/products/vpn/includes/featured-press.html

```diff
@@ -1,15 +0,0 @@
-{#
- This Source Code Form is subject to the terms of the Mozilla Public
- License, v. 2.0. If a copy of the MPL was not distributed with this
- file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-<aside class="vpn-featured-press-logos">
-  <h3 class="vpn-featured-press-heading">{{ ftl('vpn-shared-featured-in') }}</h3>
```

Comment:
**question (non-blocking, l10n):** if we were quite certain this block was not coming back, would we remove the ftl translation? or just mark is as outdated?

Reply:
The block might come back, so I'd say we should keep the strings for now